### ISSUE TITLE
VuePress Migration Feedback

### ISSUE THREAD

**Issue by bencodezen** [1950Labs/vuejs.org#26]
Hello @sxcamacho and other others I might have missed!

My name is Ben Hong and I'm a Vue.js Community Partner that is currently working with the Vue.js Core Docs Team to help with updating the platform from Hexo to VuePress. During this time, we plan to have a content freeze in order to ensure all of the sites stay in sync with one another.

As maintainers for the Spanish translation site, are there any pain points or concerns that you might have about this migration? We would love to hear from you and make sure your thoughts are taken into consideration before starting work on this.

Looking forward to your response!

**Comment by sxcamacho** [comment]
Hey @bencodezen , it's a pleasure to be in touch with you, thank you for keeping us in mind. 
It's very cool to know that you are planing migrate to VuePress!

About to the migration we have some questions:

1. When could we have the English site in VuePress?
2. The idea is to take the new English site in VuePress and work on the translation in the same way as in the current site? Creating a clone and translating all the content?
3. Do you know what is the best way to stay in sync with the latest changes made on the English site? keeping tuned to the latest commits and translating these?

**Comment by leonelmore** [comment]
Hi @bencodezen, I think it would be great if you can share a roadmap/milestones with dates for this change to sync the translation efforts.

How do you plan to deal with the not updated content (to VuePress) during the transition? Do you see both sites working altogheter during this time?

Another concern I have is what can we do to prepare for this change, having in mind we are still working on translating content.

Thanks again!

**Comment by bencodezen** [comment]
@sxcamacho Thanks for the quick response! As far as your questions go:

1. We are currently in the planning stage and will let you know when plan to formally freeze the content with launch dates so you are kept up to date on what's going on.
2. Correct. Everything should stay the same as far as hosting and domains go. We're hoping to make the migration as painless as possible for translators.
3. This is a great question. I will bring this up with the team!

**Comment by bencodezen** [comment]
@leonelmore Thank you for sharing your concerns! Here are our thoughts on the points that you raised:

We are still working on the roadmap for the migration. Our goal is to have that wrapped up soon so we can share the plan with everyone, but we wanted to allow translators to weigh in on things beforehand in case we made any incorrect assumptions.

During the transition, we would tell everyone that a content freeze will be happening. So no updated content during the transition (which we want to complete within a small time frame). 

As far as preparing for the change, the main thing might just be familiarizing yourself with VuePress, but we plan on making the migration as simple as possible since we still want the authoring process to be great for everyone.

Let me know if you have other questions or need further clarification!

**Comment by FLarra** [comment]
Hi everyone!

Thanks for the heads up @bencodezen  . It's great to hear that you're planning to move the documentation over VuePress.

I have a few questions too:
1. From my perspective, it would be nice to have the Spanish translation public before moving the "Spanish repo" over VuePress so people can start getting advantage of it with the documentation we already have. What do you think? May that affect on something from your side?

2. If # 1 is reasonable, I think we could also benefit from you since you will be ahead on the migration and the problems you face would be easier for us to solve, since we will know from you how to solve them. In this case, would be great if you can share the obstacles and learnings so we don't reinvent the wheel.

I'm really excited about this! 😃

**Comment by bencodezen** [comment]
@FLarra Yes we don't plan on taking down any of the current translations at the moment. The main plan is to have a content freeze as far as no new updates for the English content to make it easier for everyone.

We will definitely be writing some sort of migration guide for translators to help with the process.

Does that help to answer your questions?

**Comment by FLarra** [comment]
@bencodezen Kind of.. haha. Currently the Spanish documentation is not live yet but I think we are pretty close to finishing it* (the first version). 

What I was trying to say is that it would be valuable (from my point of vue) to have the Spanish version published and then start the spanish migration to VuePress. What do you think?

**Comment by bencodezen** [comment]
@FLarra I totally agree with you! Go ahead and publish the Spanish site right whenever you're ready. No need to wait for the migration at all. 😄 

We'll be sure to keep you posted whenever we have formally migrated the English one successfully and then send a migration guide as needed.

**Comment by bencodezen** [comment]
If anyone has any additional feedback, please don't hesitate to re-open the issue and/or tag me in a new one!
